README.md

```diff
@@ -4,9 +4,22 @@ ramda-fantasy
 [Fantasy Land][1] compatible types for easy integration with [Ramda][2].
 
 ## Project status
-This project is in alpha status. The implementation of the Fantasy Land spec should be *mostly*
-stable. Any methods outside of the Fantasy Land spec are subject to change. The types also have
-not undergone thorough testing/use yet.
+The Ramda team has decided that this library is no longer serving the JavaScript FP community well. It has not kept up with changes to the Fantasy Land spec, and many of the data types lack useful features available in other libraries.
```

Comment:
I think it is more a matter of there are a number of alternatives in the same space. The goal was never "serving the JavaScript FP community". I think this whole paragraph can be struck